planetiler-custommap/src/main/java/com/onthegomap/planetiler/custommap/configschema/FeatureLayer.java

```diff
@@ -1,8 +1,10 @@
 package com.onthegomap.planetiler.custommap.configschema;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.Collection;
 
 public record FeatureLayer(
   String id,
-  Collection<FeatureItem> features
+  Collection<FeatureItem> features,
+  @JsonProperty("tile_post_process") PostProcess postProcess
```

Comment:
The change looks good! Only thing left is to mention these features in `planetiler.schema.json` and `planetiler-custommap/README.md`

Reply:
Change pushed.